### ISSUE TITLE
cannot install on windows: UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 262: character maps to <undefined>

### ISSUE THREAD

**Issue by talebi1** [marktennyson/nc-console#1]
```cmd
× Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [18 lines of output]
      Traceback (most recent call last):
        File "c:\temp\venv\lib\site-packages\pip\_vendor\pep517\in_process\_in_process.py", line 363, in <module>
          main()
        File "c:\temp\venv\lib\site-packages\pip\_vendor\pep517\in_process\_in_process.py", line 345, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
        File "c:\temp\venv\lib\site-packages\pip\_vendor\pep517\in_process\_in_process.py", line 130, in get_requires_for_build_wheel
          return hook(config_settings)
        File "C:\Users\toto\AppData\Local\Temp\pip-build-env-jjxteo8s\overlay\Lib\site-packages\setuptools\build_meta.py", line 177, in get_requires_for_build_wheel
          return self._get_build_requires(
        File "C:\Users\toto\AppData\Local\Temp\pip-build-env-jjxteo8s\overlay\Lib\site-packages\setuptools\build_meta.py", line 159, in _get_build_requires
          self.run_setup()
        File "C:\Users\toto\AppData\Local\Temp\pip-build-env-jjxteo8s\overlay\Lib\site-packages\setuptools\build_meta.py", line 174, in run_setup
          exec(compile(code, __file__, 'exec'), locals())
        File "setup.py", line 8, in <module>
          long_description = f.read()
        File "C:\Python39\lib\encodings\cp1252.py", line 23, in decode
          return codecs.charmap_decode(input,self.errors,decoding_table)[0]
      UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 262: character maps to <undefined>
      [end of output]
```
The issue is caused by the big ascii NCCONSOLE in README.md, please remove it or replace it with something utf-8 friendly.

**Comment by marktennyson** [comment]
Thanks, @talebi1 for this issue.
This issue is fixed on the new release => `0.1.0`

**Comment by talebi1** [comment]
> Thanks, @talebi1 for this issue. This issue is fixed on the new release => `0.1.0`

awesome, thank you @marktennyson
